Lib/pdb.py

```diff
@@ -840,6 +843,17 @@ def checkline(self, filename, lineno):
             return 0
         return lineno
 
+    def checkexpr(self, expr):
```

Comment:
At the moment. This might change in the future, at which point it would be a breaking change to make it a regular method, as some users would be calling it from the class object instead of an instance. So I don't think we should mark the method as a static method.

In fact, in Python there is little reason to use static methods at all (unlike class methods) because you can just define module-level functions instead.